app.rb

```diff
@@ -0,0 +1,175 @@
+require_relative 'app_helper'
+require 'json'
+class App
+  def initialize
+    @books = []
+    @labels = []
+    @authors = []
+    @games = []
+    @music_albums = []
+    @genres = []
+  end
+
+  # rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] Just as pointed out above, As required by Microverse, you are not meant to use any linters disable rule on your codebase. I suggest you remove this and try to fix the issue  :+1: :100: